src/pip/_internal/basecommand.py

```diff
@@ -313,16 +313,24 @@ def populate_requirement_set(requirement_set, args, options, finder,
             if options.find_links:
                 raise CommandError(
                     'You must give at least one requirement to %(name)s '
-                    '(maybe you meant "pip %(name)s %(links)s"?)' %
-                    dict(opts, links=' '.join(options.find_links)))
+                    '(maybe you meant "pip %(name)s %(links)s"?)'
+                    % dict(opts, links=' '.join(options.find_links))
+                )
             else:
                 raise CommandError(
                     'You must give at least one requirement to %(name)s '
-                    '(see "pip help %(name)s")' % opts)
+                    '(see "pip help %(name)s")' % opts
+                )
 
-    def _build_package_finder(self, options, session,
-                              platform=None, python_versions=None,
-                              abi=None, implementation=None):
+    def _build_package_finder(
+        self,
+        options,
+        session,
+        platform=None,
+        python_versions=None,
+        abi=None,
+        implementation=None,
+    ):
```

Comment:
I dislike this style for function definitions. This is the second time this rule has been applied, so it's obviously a black style rule, but I prefer our existing format,.